.pre-commit-config.yaml

```diff
@@ -33,3 +33,4 @@ repos:
           - pytest-vcr
           - types-requests
           - types-python-dateutil
+          - orjson
```

Comment:
At a glance, I don't see any code in this PR that would need this change -- you sure it's required?